client/docs/swagger-ui/swagger.yaml

```diff
@@ -33370,12 +33329,53 @@ paths:
           in: path
           required: true
           type: string
-        - name: query_data
-          description: QueryData contains the query data passed to the contract
-          in: path
-          required: true
+        - name: pagination.key
+          description: |-
+            key is a value returned in PageResponse.next_key to begin
+            querying the next page most efficiently. Only one of offset or key
+            should be set.
+          in: query
+          required: false
           type: string
           format: byte
+        - name: pagination.offset
+          description: >-
+            offset is a numeric offset that can be used when key is unavailable.
+
+            It is less efficient than using key. Only one of offset or key
+            should
+
```

Comment:
Unnecessary line breaks in a sentence?

Reply:
`swagger.yaml` file is generated by `make proto-swagger-gen`.
So, we don't need to modify swagger file.